.github/workflows/ci-workflow.yml

```diff
@@ -6,7 +6,16 @@ name: github-linux
 ############################## TRIGGERS ##############################
 # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#on
 # This workflow will be triggered on git action defined below e.g. push, pull_request.
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+    paths-ignore:
```

Comment:
Maybe we could add `paths-ignore` to push action as well, because now it seems that it only applies to the `pull request` action?

Reply:
Thanks @marcinwrobel1986 for catching this, fixed on 3e8a5108b8370f393ec3e518565ee35dcfc00f8c

@fnrizzi @marcinwrobel1986
Can workflows be triggered for PRs and commits to `main` and `develop` ? This way we cover regular PRs - also out of forks (with `synchronize`) - and direct, quick fixes. This policy may trigger duplicated runs on merges, I'm not sure.

Reply:
we already have a guard for duplicates